### DISCUSSION TITLE
Does anyone have an example of using FastAPI with `mint`??

### DISCUSSION THREAD

**Discussion by dan-jacobson** [mintoolkit/mint#58] [Q&A]
I'm banging my way through trying to get `mint` to work with a `fastAPI` image I have. A lot of the existing `python` examples are too old to be very useful for me. Would love to see an example if anyone has one!

Otherwise, I'll try to contribute back a minimal example if I can figure it out.

**Comment by kcq** [comment]
@dan-jacobson there's a couple of FastAPI examples in the 3rdparty directory: https://github.com/mintoolkit/examples/tree/master/3rdparty/python_fastapi_official and https://github.com/mintoolkit/examples/tree/master/3rdparty/python_poetry_fastapi_svx

Do you have any kind of websocket or file upload logic in your app?

**Reply by dan-jacobson** [reply]
This is awesome! Exactly what I was looking for :). I scrolled through the other python examples but didn't think to look in `3rdparty`.

Yup, I have file upload logic in my app, so gotta find the correct `--http-probe-cmd-upload` to get that working. Thank you!
